python/meterpreter/meterpreter.py

```diff
@@ -483,7 +483,7 @@ def get_system_arch():
         ctypes.windll.kernel32.GetNativeSystemInfo(ctypes.byref(sysinfo))
         values = {0:'x86', 5:'armle', 6:'IA64', 9:'x64'}
         arch = values.get(sysinfo.wProcessorArchitecture, uname_info[4])
-    if arch == 'x86_64':
+    if arch == 'x86_64' or arch.lower() == 'amd64':
```

Comment:
https://github.com/rapid7/metasploit-payloads/pull/555#issuecomment-1077720125

In partiular:

> Cygwin on Windows uses "x86_64", FreeBSD uses "amd64", Linux uses "x86_64". Mac OS X uses "x86_64" for Intels (and ppc64" for PowerPCs).

Should we add those values too? 👀